packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -480,7 +480,7 @@ addFieldsDict(Posts, {
         sequence = await Sequences.loader.load(post.canonicalSequenceId);
       }
 
-      return accessFilterSingle(currentUser, Sequences, sequence);
+      return sequence ? accessFilterSingle(currentUser, Sequences, sequence) : null
```

Comment:
`accessFilterSingle` already checks for null and handles it by returning null, so this is redundant